README.md

```diff
@@ -192,7 +191,6 @@ If you have questions or comments, please create an issue.
 > List of initiatives that aggregate beginner friendly issues to work on or seasonal events.
 
 - [Up For Grabs](https://up-for-grabs.net/) - Contains projects with beginner-friendly issues.
-- [First Timers Only](https://www.firsttimersonly.com/) - A list of bugs that are labelled "first-timers-only".
```

Comment:
This can be kept and added back in.

Reply:
done https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/cf4b2cc817a90fd08f596925eecea5e29f070bda